components/lsp-form/LSPForm.tsx

```diff
@@ -87,12 +132,48 @@ const LSPForm: FC<Props> = ({
       {showSettle && (
         <SettleWrapper>
           <SettleTitle>Settle Position</SettleTitle>
-          <SettleText>
-            The LSP contract has expired. You can now settle your position at
-            the oracle returned price.
-          </SettleText>
-          <SettleButton onClick={() => setShowSettle(false)}>
-            Settle
+          {contractState === ContractState.Open && (
+            <SettleText>
+              The LSP contract is expireable. You can now settle your position
+              at the oracle returned price.
+            </SettleText>
+          )}
+          {/* contractState is an enum -- when it's not open, it's greater than 0. */}
+          {contractState === ContractState.ExpiredPriceRequested && (
+            <SettleText>
+              The LSP contract is expired. The final price most go through the
+              Optimistic Oracle first -- once this price is settled, you can
+              redeem your tokens. Please check back later.
+            </SettleText>
+          )}
+          {contractState === ContractState.ExpiredPriceReceived && (
+            <SettleText>
+              The LSP contract is expired. You can now settle your position at
+              the oracle returned price. The following tokens will be redeemed:
+              <SettleTokenBalance>
+                Long Token Balance:{" "}
+                {ethers.utils.formatUnits(longTokenBalance, collateralDecimals)}
+              </SettleTokenBalance>
+              <SettleTokenBalance>
+                Short Token Balance:{" "}
+                {ethers.utils.formatUnits(
+                  shortTokenBalance,
+                  collateralDecimals
+                )}
```

Comment:
we might want a way to show the user how much collateral they will be recieving based on price @Tommy1231232